server/index.ts

```diff
@@ -17,7 +17,12 @@ const assistantConfig = {
   schema: schema.object({
     chat: schema.object({
       enabled: schema.boolean({ defaultValue: false }),
-      rootAgentName: schema.maybe(schema.string()),
+      rootAgentName: schema.conditional(
```

Comment:
Would be nice to add a doc entry for `rootAgentName` in [GETTING_STARTED_GUIDE.md](https://github.com/opensearch-project/dashboards-assistant/blob/main/GETTING_STARTED_GUIDE.md)